src/java/org/apache/cassandra/metrics/MemtableFlushMetrics.java

```diff
@@ -0,0 +1,38 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package org.apache.cassandra.metrics;
+
+import com.codahale.metrics.Gauge;
+import org.apache.cassandra.utils.memory.MemtableCleanerThread;
+
+import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
+
+public class MemtableFlushMetrics
```

Comment:
@ekaterinadimitrova2 What would you say to just putting this in `TableMetrics` with the other Memtable-realted metrics? We'd have to qualify the name a bit, but something like "MemtablePendingFlushTasks", "MemtablePendingFlushes", etc. might work.

Reply:
If I'm right the metrics in `TableMetrics` are for a specific table, and there is also an `allTableMetrics` aggregating all of them. However, the new memtable flush metric is global, so it can't be associated to a single table. Perhaps we can put the new metric under `MemtablePool`, where `BlockedOnAllocation` lives?

Also, `MemtableFlushMetrics` is instantiated only by the test, so I think it isn't available when running Cassandra itself.

Reply:
> the metrics in TableMetrics are for a specific table

Ouch, you're right.

> Perhaps we can put the new metric under MemtablePool, where BlockedOnAllocation lives?

+1